content/releases/3-0-0rc08.md

```diff
@@ -0,0 +1,167 @@
+
+**Fix**
+- Fix dgraph hanging on shutdown request
+
+**Remove**
+- Remove wait groups for Enterprise Dgraph ACL functionality
+- Remove license scanning CI/CD job
+
+### BPMN
+
+**Add**
+- Add log message and time delay to CallActivity watcher
+- Add graceful shutdown to command consumer port
+- Add environmental variable expansion to json-schema service task
+- Add Async Publish Error logging to NATS KVs
+- Add profile labels to go-routine launches
+- Add multi-file JSON schema validation
+
+**Change**
+- Change InProgess message to 20s on CommandConsumer from 29s
+- Change debug level log messages for timer checks ands active workflows to trace level
+- Change log level of gateways without inputs to trace level from error
+- Change "libreBpmn.command." strings to use domain constant
+- Change CallActivity to event driven as opposed to a blocking go-routine to wait for complete of a sync call
+- Change to git commits to use LN on *.go files
+
+**Fix**
+- Fix NATS reconnect re-subscribing to startOnNATS Topics
+- Fix BPMN long save times by only updating the touched Workflow Specification
+- Fix docker permissions in end-to-end CI/CD test case
+- Fix panic on nil workflowspec in HandleTaskComplete
+
+### Core
+
+**Features**
+- Add operations event definition versioning mutations
+- Add agent MQTT message handling
+- Add option to activate newly created version if requested
+- Add check for empty migration records
+- Add label to OperationsEventClassProperty, OperationsEventDefinitionProperty, OperationsEventProperty
+- Add initialization for Azure stream
+- Add option for Azure data source type
+- Add binding path test cases
+- Add debug logging to updateOperationsEventRecordEntry
+- Add entity path to migration dependency checks
+- Add `hierarchyScope`, `materialAlternate`, `spatialDefinition` and `unitOfMeasure` as information objects to `GetOperationsEvent` `operationsEventRecords` 
+- Add check for migration dependencies on Operations Event Record Entry 
+
+**Change**
+- Change IncludesPropertiesOf to be on the version not the header of OperationsEventDefinition & OperationsEventClass
+- Change async SaveVersionAs because other cases have been tested in sync tests
+- Change Equipment Class rule triggered event to immediately publish to NATS and then be picked up by Core instead of waiting an triggering to prevent libre-core shutdowns missing the event fire
+- Change consumer creation to delete/add consumer if it fails to update consumer
+- Change logging messages to reflect data source type
+- Change default logger to use hostname instead of PID
+- Change database ping to allow `no access token provided` and `context cancelled` when pinging database
+- Change migrations to remove existing children before checking for migration dependencies
+- Change Operations Event Record Entry migration to remove existing children before checking for migration dependencies
+
+**Remove**
+- Remove obsolete comments
+

```

Comment:
```suggestion
- Change `IncludesPropertiesOf` to be on the version not the header of `OperationsEventDefinition` & `OperationsEventClass`
- Change async `SaveVersionAs` because other cases have been tested in sync tests
- Change Equipment Class rule triggered event to immediately publish to NATS and then be picked up by Core instead of waiting an triggering to prevent libre-core shutdowns missing the event fire
- Change consumer creation to delete/add consumer if it fails to update consumer
- Change logging messages to reflect data source type
- Change default logger to use `hostname` instead of PID
- Change database ping to allow `no access token provided` and `context cancelled` when pinging database
- Change migrations to remove existing children before checking for migration dependencies
- Change Operations Event Record Entry migration to remove existing children before checking for migration dependencies

**Remove**
- Remove obsolete comments


```